app/ldap.js

```diff
@@ -564,6 +564,11 @@ exports.addGroup = function (group, cb) {
   });
 };
 
+// check if LDAP is disabled
+exports.isDisabled = function() {
+  return (!process.env.NODE_ENV || 'development' === process.env.NODE_ENV) && 'true' !== process.env.LDAP;
```

Comment:
Change line 568 to: 
```javascript
return (!'development' === process.env.NODE_ENV) && 'true' !== process.env.LDAP;
````

The first check is not needed. Other than that I can accept this since your tests actually pass :)